Core/src/Material/VolumeMaterialMapper.cpp

```diff
@@ -292,7 +292,7 @@ void Acts::VolumeMaterialMapper::mapMaterialTrack(
       double distVol = (volIter->position - mTrack.first.first).norm();
       double distMat = (rmIter->position - mTrack.first.first).norm();
       // Material past the entry point to the current volume
-      if (distMat > distVol) {
+      if (distMat - distVol > s_epsilon) {
         // Switch to next material volume
```

Comment:
same question here, could a `std::abs()` be needed?

Reply:
Same applies then I guess.

Reply:
Yes it is the exact same think as the other one, the one in the surface mapper is used to make sure we don't map material which is in a material volume to surface and the one in the volume mapper is used to decide onto which volume we map the material.